datasets/logiqa_en/README.md

```diff
@@ -0,0 +1,151 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: https://github.com/huggingface/datasets-tagging
+---
+
+# Dataset Card for LogiQA
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+- **Repository: https://github.com/lgw863/LogiQA-dataset**
+- **Paper: https://arxiv.org/pdf/2007.08124.pdf**
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+LogiQA, which contains 8,678 paragraph-question pairs, each with four candidate answers. The dataset is sourced from publically available logical examination papers for reading comprehension, which are designed by domain experts for evaluating the logical reasoning ability and test participants.
+
+### Supported Tasks and Leaderboards
+
+Question-Answering
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+The each example consists of 8 lines from the text files of a given split.
+These lines consists of strings and constitute a single question-answer pair with a following context and set of 4 multiple choice answers.
+Here is an example:
+```buildoutcfg
+c
+Some Cantonese don't like chili, so some southerners don't like chili.
+Which of the following can guarantee the above argument?
+A.Some Cantonese love chili
+B.Some people who like peppers are southerners
+C.All Cantonese are southerners
+D.Some Cantonese like neither peppers nor sweets
+
+```
```

Comment:
The **Data Instances** section is intended to be used to display an actual data point, in JSON format. In this case that would be:
```JSON
{
"answer": "c",
"context": "Some Cantonese don't like chili, so some southerners don't like chili.",
"question": "Which of the following can guarantee the above argument?",
"choice_a": "Some Cantonese love chili",
"choice_b": "Some people who like peppers are southerners",
"choice_c": "All Cantonese are southerners",
"choice_d": "Some Cantonese like neither peppers nor sweets"
}
```

Reply:
Should this still be the format even though the actual data is in a text format and delimited by line?

Reply:
The dataset card (README file) is intended to describe the format of the data as yielded by the 🤗 `datasets` library, so yes !

Reply:
Ah I see. Done!